particle/shared_literals.py

```diff
@@ -0,0 +1,86 @@
+# Licensed under a 3-clause BSD style license, see LICENSE.
+"""
+Helper module with common particle name aliases (PDGID class instances).
+"""
+
+from __future__ import absolute_import
+
+# Names have to be valid identifiers.
+# Values can be repeated.
+
+common_particles = dict(
```

Comment:
I don't understand the need to define `common_particles` dict. https://github.com/scikit-hep/particle/blob/master/particle/data/mass_width_2018.mcd basically has a mapping from PDGID to names. I think it is ok to provide some aliases in addition, like it is done here (photon = gamma = 22). But that can be done after generating the bulk of the names from the table.

Reply:
Seems we are talking about 2 slightly different things: this file gives us more flexibility whereas what you refer to seems more something that ca be queried programmatically via the Particle class?

Having an intermediate set will also make it easer if/when the PDG goes away from their mcd files once their API is out. We have full control here, albeit with some more work - not much on the grand scheme of things.

Let me think about this a bit more ...

Reply:
BTW, having the dict is also a neat way to find out what is actually defined as aliases, with some structure. The mcd file is not as friendly.

Reply:
I worry about keeping a consistent mapping between names and ids. I think the literals should not be named by hand, as is currently done in `common_particles`, I think the information should come from an externally curated - ideally stable - source. If the PDG  moves away from mcd files, but keep the same naming conventions, then it would be ok to pull the names from the mcd files. In the worst case, a new reader for the new data file format has to be written.